### ISSUE TITLE
[UI idea] Readable text on mobile layouts

### ISSUE THREAD

**Issue by arjank** [Felienne/hedy#3379]
**The text should be readable on all devices, including mobile devices**
Currently the text is not really readable on mobile devices, it's too small.

The text on this example image appears bigger than it does in reality. This issue isn't about making things perfect, the main objective is to get a decent starting point that we can use for further improvements.

![image](https://user-images.githubusercontent.com/457006/193900174-5ef9229e-99ab-405a-ac5f-56823bdf2e51.png)

**Comment by TiBiBa** [comment]
I like that this issue is raised, however I think that when wanting a well-implemented mobile experience we should implement two seperate UIs.

We simply have too much elements and dependencies to fix this with css to perform user-friendly on pc and mobile. This, of course, is a (very) big project so this is a good starting point for the time-being. 

But I think we should be wary to invest to much time in tweaking our current UI to be mobile-friendly instead of starting a separate one from scratch.

**Comment by arjank** [comment]
I totally agree that we shouldn't invest too much time, at least without a proper plan on what the mobile layout should be. I'm not sure it should be a completely different layout, but it should be designed. Then again, the main purpose of this ticket was to update the current layout to make it readable on mobile without having to zoom in manually. That's a one line fix. That means there was time left to check some pages and improve some other pages.
